pgml-dashboard/src/templates/mod.rs

```diff
@@ -138,70 +141,66 @@ impl<'a> WebAppBase<'a> {
     }
 
     pub fn nav(&mut self, active: &str) -> &mut Self {
-        let mut nav_links = vec![NavLink::new("Create new cluster", "/clusters/new").icon("add")];
-
-        // Adds the spesific cluster to a sublist.
-        if self.visible_clusters.len() > 0 {
-            let mut sorted_clusters: Vec<(String, String)> = self
-                .visible_clusters
-                .iter()
-                .map(|(name, id)| (name.to_string(), id.to_string()))
-                .collect();
-            sorted_clusters.sort_by_key(|k| k.1.to_owned());
-
-            let cluster_links = sorted_clusters
-                .iter()
-                .map(|(name, id)| {
-                    NavLink::new(name, &format!("/clusters/{}", id)).icon("developer_board")
-                })
-                .collect();
-
-            let cluster_nav = Nav {
-                links: cluster_links,
-            };
-
-            nav_links.push(
-                NavLink::new("Clusters", "/clusters")
-                    .icon("lan")
-                    .nav(cluster_nav),
-            )
-        } else {
-            nav_links.push(NavLink::new("Clusters", "/clusters").icon("lan"))
+        let (upper_nav_links, lower_nav_links) = match config::standalone_dashboard() {
+            true => (
+                vec![
+                    // NavLink::new("Notebooks", "/notebooks")
+                    //     .icon("description")
+                    //     .disable(true),
+                    // NavLink::new("Explore", "/explore")
+                    //     .icon("thumbnail_bar")
+                    //     .disable(true),
+                    // NavLink::new("Projects", "/projects")
+                    //     .icon("library_add")
+                    //     .disable(true),
+                    NavLink::new("Dashboard", "/dashboard").icon("thumbnail_bar"),
+                ],
+                vec![],
+            ),
+            false => (
+                vec![
+                    // NavLink::new("Notebooks", "/notebooks")
+                    //     .icon("description")
+                    //     .disable(true),
+                    // NavLink::new("Explore", "/explore")
+                    //     .icon("thumbnail_bar")
+                    //     .disable(true),
+                    // NavLink::new("Projects", "/projects")
+                    //     .icon("library_add")
+                    //     .disable(true),
+                    NavLink::new("Status", &format!("/clusters/{}", self.current_cluster.id))
```

Comment:
At some point, we would like to be able to pass these in from our cloud app, because the URLs and names could change, but for now, this will work.

Reply:
I changed this so the links are in context and set in guards and fairings.